_posts/2021-04-14-cockpit-242.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Cockpit 242
+author: marusak
+date: 2021-04-14
+tags: cockpit linux
+slug: cockpit-242
+category: release
+summary: Cockpit ...
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 242.
+
+
+## Support for pages built with snowpack
```

Comment:
Perhaps one of these instead:

```suggestion
## Initial Snowpack support for pages
```

```suggestion
## Groundwork for Snowpack support
```

Reply:
Snowpack is a proper name of a project, so it should be capitalized, unless the project decides to invoke custom stylistic rules. (Note: They haven't.)

Reply:
Of course, if you're referring to the command `snowpack`, that's different. But for the headline, it should be talking about Snowpack as a whole, not the specific command to invoke it.

Reply:
I did mean the project name, so capitalizing it sounds appropriate. Its website uses "Snowpack".